src/simulation_models/stage_update.jl

```diff
@@ -130,7 +130,7 @@ function _calculate_ic_quantity(initial_condition_key::ICKey{TimeDurationON, PSD
 
     current_counter = time_cache[:count]
     last_status = time_cache[:status]
-    var_status = var_value > eps() ? 1.0 : 0.0
+    var_status = isapprox(var_value, 0.0, atol = 1e-4) ? 1.0 : 0.0
```

Comment:
And why not use the binary status variable value here? instead of using the actual output variable ?
This is not clear to me, may be I'm missing something here.

Reply:
Because the binary won't always be available if the FF is from a dispatch to a dispatch